website/src/data/users.js

```diff
@@ -559,6 +559,14 @@ const users = [
     source: 'https://github.com/facebook/react-native-website',
     tags: ['facebook', 'large', 'highlight', 'design', 'versioning'],
   },
+  {
+    title: 'React Native Boilerplate',
+    description: 'A React Native project template for building solid applications through separation of concerns between the UI, state management and business logic.',
+    preview: require('./showcase/reactnativeboilerplate.png'),
+    website: 'https://thecodingmachine.github.io/react-native-boilerplate/',
+    source: 'https://github.com/thecodingmachine/react-native-boilerplate',
+    tags: ['thecodingmachine', 'redux', 'react-native', 'architecture', 'fastlane', 'boilerplates', 'react-native-boilerplate', 'redux-toolkit'],
```

Comment:
```suggestion
    tags: [],
```

removing tags, we assign tags ;)